android/libraries/rib-base/src/androidTest/java/com/badoo/ribs/test/util/AndroidTestUtils.kt

```diff
@@ -0,0 +1,28 @@
+package com.badoo.ribs.test.util
+
+import android.app.Activity
+import android.support.test.InstrumentationRegistry
+import android.support.test.rule.ActivityTestRule
+import java.util.concurrent.TimeoutException
+
+fun Activity.waitForDestroy() {
+    waitFor { isDestroyed }
+}
+
+fun <T: Activity> ActivityTestRule<T>.waitForActivityFinish() {
+    val activity = this.activity
+    finishActivity()
+    activity.waitForDestroy()
+    InstrumentationRegistry.getInstrumentation().waitForIdleSync()
+}
+
+fun waitFor(timeoutMillis: Long = 1000L, condition: () -> Boolean) {
```

Comment:
After how many times I saw it failing on your laptop with only 1000L, maybe consider a better default value :)

Reply:
➕ 
also, we have super-slow ARM emulators on CI

Reply:
It was failing only on debug
@lukaville I think 1 second is more than enough even for slow emulators?

Reply:
@ShikaSD
There are no guarantees that it will be executed in 1 second. We had many flaky tests due to low timeouts and it's always not enough :) I think we need to increase it to max value that is acceptable to wait in case of real issue with the business logic. imho 10 seconds is enough

Reply:
Increased to 10s.